sql/000-ore.sql

```diff
@@ -239,13 +239,27 @@ RETURNS integer AS $$
   DECLARE
     cmp_result integer;
   BEGIN
-    IF (array_length(a, 1) = 0 OR a IS NULL) AND (array_length(b, 1) = 0 OR b IS NULL) THEN
+    IF a IS NULL AND b IS NULL THEN
+      RETURN NULL;
+    END IF;
```

Comment:
Hmm, could this be `IF a IS NULL OR b IS NULL`, and get rid of `IF a IS...` and `IF b IS...` below?

Reply:
Yes. Lol. Thank you